charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.9.3
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.52.0
+version: 5.53.0
```

Comment:
Lets go for a major bump and completely fulfill the Semver spec ;) we don't need another project doing Semver with a custom deprecation policy 🙈

Reply:
Bumped